plugins/justify/plugin.js

```diff
@@ -178,8 +188,18 @@
 
 		refresh: function( editor, path ) {
 			var firstBlock = path.block || path.blockLimit;
-
-			this.setState( firstBlock.getName() != 'body' && getAlignment( firstBlock, this.editor.config.useComputedState ) == this.value ? CKEDITOR.TRISTATE_ON : CKEDITOR.TRISTATE_OFF );
+			var name = firstBlock.getName();
+
+			// Check if we can give to element proper class, style or current path is body (#16808).
+			// IE8, after focus and refresh return path with 'body'
+			if ( ( this.cssClassName && editor.activeFilter.check( name+'('+this.cssClassName+')' ) ) ||
+				editor.activeFilter.check( name+'{text-align}' ) ||
+				name == 'body' )
```

Comment:
@Comandeer there are OR statements so there won't be always `false`. If there will be allowed element which can have style or class then condition will be `true`.
Do I understand you correctly that I should change 3rd condition from `name == 'body'` to `firstElement == editor.editable()`?
If not then please explain a little bit better what do . you mean by:
> Therefore I'd check if the `firstBlock` is not the same node as editable.